plugins/pom.xml

```diff
@@ -214,6 +214,12 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
```

Comment:
I think that we could add this dependency in the parent pom, it could be useful also for tests in core.

Reply:
I considered adding it higher in the hierarchy, but components parent doesn't use it and ui-bundles doesn't use it either, both projects don't have non-provided dependencies. For all effects, this pom is the root of all component plugins.